assignments/1. hello-world/external/index.html

```diff
@@ -6,5 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Internal</title>
   </head>
-  <body></body>
+  <body>
+    <script src="main.js"></script>
```

Comment:
It's a good practice to use `./` in front of the file name you are linking. Like in this case use `./main.js`